sdk/purview/Azure.Analytics.Purview.Sharing/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Looks like 1.0.3-beta.20 was indeed already out. Please change the release version that Jesse recommended.

Reply:
@jsquire @pallavit  We have 1.0.3-beta.20 out for Azure.Analytics.Purview.Share. We were given feedback on the last review to rename it to .Sharing instead of .Share. With that being said, does 1.0.3-beta.20 make sense here? Or would you suggest a different value?